challenges/objects-arrays.js

```diff
@@ -90,7 +154,16 @@ const zooAnimals = [
 The zoos want to display both the scientific name and the animal name in front of the habitats. Populate the displayNames array with only the animal_name and scientific_name of each animal. displayNames will be an array of strings, and each string should follow this pattern: "Name: Jackal, asiatic, Scientific: Canis aureus."
 
 */
+
+function print(item) {
+  console.log(`Name: ${item.animal_name},  Scientific: ${item.scientific_name}`);
+}
+
 const displayNames = [];
+function findNames(item) {
+ return displayNames.forEach(item); 
```

Comment:
While this code is close, you're not calling forEach on the array, also remember that forEach does not automatically create a new array.